content/applications/finance/accounting/receivables/customer_invoices/epc_qr_code.rst

```diff
@@ -1,68 +1,58 @@
-it greatly reduces typing errors that would potentially
-make for payment issues.
+In addition to bringing ease of use and speed, it greatly reduces typing errors that would
+potentially make for payment issues.
 
 .. note::
-   This feature is only available in several European
-   countries such as Austria, Belgium, Finland, Germany,
-   and The Netherlands.
+   This feature is only available in several European countries such as Austria, Belgium, Finland,
+   Germany, and The Netherlands.
 
 Configuration
 =============
 
-Go to :menuselection:`Accounting --> Configuration --> Settings`
-and activate the **SEPA QR Code** feature.
+Go to :menuselection:`Accounting --> Configuration --> Settings` and activate the :guilabel:`QR
+Codes` feature.
 
-.. image:: epc_qr_code/epc_qr_code01.png
-   :align: center
-
-Configure your Bank Account’s journal
+Configure your bank account’s journal
 -------------------------------------
 
-Make sure that your *Bank Account* is correctly configured
-on Odoo with your IBAN and BIC.
+Make sure that your :guilabel:`Bank Account` is correctly configured on Odoo with your IBAN and BIC.
 
-To do so, go to :menuselection:`Accounting --> Configuration
---> Journals`, open your *bank journal*, then fill out the
-*Bank Account* and *Bank* under the *Bank Account* tab.
+To do so, go to :menuselection:`Accounting --> Configuration --> Journals`, open your bank journal,
+then fill out the :guilabel:`Account Number` and :guilabel:`Bank` under the :guilabel:`Bank Account
+Number` column.
 
-.. image:: epc_qr_code/epc_qr_code02.png
-   :align: center
+.. image:: epc_qr_code/bank-journal.png
+   :alt: Bank account number column in the bank journal
 
-Issue Invoices with EPC QR Codes
+Issue invoices with EPC QR codes
 ================================
 
-EPC QR Codes are added automatically to your invoices, as
-long as you issue them to customers that are located in a
-country where this feature is available.
+EPC QR codes are added automatically to your invoices, as long as you issue them to customers that
+are located in a country where this feature is available.
+
+Go to :menuselection:`Accounting --> Customers --> Invoices`, and create a new invoice.
 
-Go to :menuselection:`Accounting --> Customers --> Invoices`,
-and create a new invoice.
+Before posting it, open the :guilabel:`Other Info` tab. Odoo automatically fills out the
+:guilabel:`Bank Account` field with your IBAN.
 
-Before posting it, open the *Other Info* tab. Odoo automatically
-fills out the *Bank Account* field with your IBAN.
+Make sure that the account indicated is the one you want to use to receive your customer’s payment
+as Odoo uses this field to generate the EPC QR code.
 
-Make sure that the account indicated is the one you want to use to
-receive your customer’s payment as Odoo uses this field to generate
-the EPC QR Code.
+When the invoice is printed or previewed, the QR code will be included at the bottom.
```

Comment:
Just so we can remember for later: we realized it wasn't showing for you because you were using an American company instead of Belgian. I'll add a note to make sure others don't make the same mistake